### ISSUE TITLE
Swift Mailer not authenticated for Gmail, but says email was sent

### ISSUE THREAD

**Issue by CrandellWS** [uvdesk/community-skeleton#508]
Basically i put in a wrong password and i had to add a `dump($e->getMessage)` command before I could figure it out...

It would seem like the system should notice it can not login into the mail server (Gmail in this case) and report it back through the admin panel. 

Or like on customer public front it says please check your in box but the emails were not getting sent when doing password reset.

I would think the system would be better for support if when the emails fail to send it is noted 

funny though after adding the dump command and then removing... it seems to be telling me swift mailer is not working... hmm 

Great now I got past the bad password only to be Blocked by Google Settings "Less secure app blocked" and I cant enable that setting for my domain due to company policy ... so ok, I guess i am setting up Gmail through SMTP

**Comment by PeopleInside** [comment]
This seems to be a duplicate of https://github.com/uvdesk/community-skeleton/issues/413

**Comment by CrandellWS** [comment]
refer to other thread and thank you for reply
